lms/views/api/blackboard/sync.py

```diff
@@ -25,28 +21,15 @@ def sync(self):
             self.request.user,
             self.request.lti_user,
             self.get_course(self.request.parsed_params["course"]["context_id"]),
-            self.group_set(),
+            self.request.parsed_params["assignment"]["group_set_id"],
             self.request.parsed_params.get("gradingStudentId"),
```

Comment:
There are a lot of args here, using `kwargs` would be helpful.

Reply:
Also, where the hell does `gradingStudentId` come from and why is it camel case?